newton/tests/test_import_usd.py

```
@@ -1078,6 +1078,205 @@ def test_h1(self):
         model = builder.finalize()
         self.verify_usdphysics_parser(asset_path, model, compare_min_max_coords=True, floating=True)
 
+    @unittest.skipUnless(USD_AVAILABLE, "Requires usd-core")
+    def test_granular_loading_flags(self):
+        """Test the granular control over sites and visual shapes loading."""
+        from pxr import Usd  # noqa: PLC0415
```

Ruff (0.14.3)

Unused `noqa` directive (non-enabled: `PLC0415`)

Remove unused `noqa` directive

(RUF100)